pkgs/development/libraries/openldap/default.nix

```diff
@@ -51,9 +64,12 @@ stdenv.mkDerivation rec {
 
     rm -rf $out/var
     rm -r libraries/*/.libs
+    rm -r contrib/slapd-modules/passwd/*/.libs
```

Comment:
Why is this necessary?

Reply:
There is a comment just above explaining it, which I'm naively following. Maybe this can be changed now that https://github.com/NixOS/patchelf/pull/98 has been merged.

Reply:
Hm, I tried removing these three lines, but ran into

> RPATH of binary /nix/store/3iwmkw16wvr5fcl7h3g49gh1jx5prsms-openldap-2.4.49/bin/ldapsearch contains a forbidden reference to /build/

So it's probably still necessary.